gammapy/maps/wcsnd.py

```diff
@@ -531,3 +542,63 @@ def make_region_mask(self, region, inside=True):
             np.logical_not(mask, out=mask)
         # TODO : update meta table to include something about the region used for mask creation?
         return WcsNDMap(geom=self.geom, data=mask, meta=self.meta)
+
+
+    def smooth(self, kernel='gauss', radius=1.0, **kwargs):
```

Comment:
I thought it would be nice to have a default value so that the function does not break in case the user does not know what smoothing to start off with, a smoothing of 1 pixel actually implies no smoothing.. 
Anyways, I have removed it now!